include/core/ib_particle.h

```diff
@@ -28,16 +28,35 @@ template <int dim>
 class IBParticle
 {
 public:
-  Point<dim> position;
-  // Translational velocity
-  Tensor<1, dim> velocity;
-  // Angular velocity
-  Tensor<1, 3> omega;
-
-  double radius;
-
-  // Pressure imposition location
-  Point<dim> pressure_location;
+    Point<dim> position;
```

Comment:
Add more documentations for the parameters. What is the meaning of last force, forces, last position, etc. You should add a paragraph that describes the content and the role of the class.